perun-core/src/main/java/cz/metacentrum/perun/core/bl/ServicesManagerBl.java

```diff
@@ -260,6 +260,17 @@
 	 */
 	void deleteService(PerunSession perunSession, Service service) throws RelationExistsException, ServiceAlreadyRemovedException;
 
+	/** Deletes the service.
+	 *
+	 * @param perunSession
+	 * @param service
+	 * @param forceFlag
+	 * @throws RelationExistsException
+	 * @throws InternalErrorException
+	 * @throws ServiceAlreadyRemovedException if there are 0 rows affected by deleting from DB
+	 */
```

Comment:
It would be nice to mention what the forceFlag does.

Reply:
Added.